pkg/config/config.go

```diff
@@ -16,6 +17,7 @@ import (
 //TODO replace to yaml
 const (
 	nsxOperatorDefaultConf = "/etc/nsx-operator/nsxop.ini"
+	vcHostCACertPath       = "/etc/vmware/wcp/tls/vmca.pem"
```

Comment:
I guess this is the default path, is there a config option for the path?

Reply:
no, ncp uses it directly.

Reply:
no